manifests/charts/istio-control/istio-discovery/values.yaml

```diff
@@ -505,7 +467,7 @@ global:
     zipkin:
       # Host:Port for reporting trace data in zipkin format. If not specified, will default to
       # zipkin service (port 9411) in the same namespace as the other istio components.
-      address: ""
+      address: "zipkin:9411"
```

Comment:
I'm pretty sure this will not work in most cases without namespace. Do we still want zipkin on by default in all cases ?

Reply:
Yeah just noticed this, should be fixed in the latest commit. As far as on by default - probably not, but I would not change it right now. Envoy added dynamic tracing config so telemetry team will add that soon(?)